src/libs/portfolio/helpers.ts

```diff
@@ -289,17 +290,23 @@ export function getUpdatedHints(
       const isTokenPreference = lowercaseNetworkPreferenceTokenAddresses.includes(lowercaseAddress)
       const isTokenInExternalAPIHints =
         lowercaseERC20HintsFromExternalAPI.includes(lowercaseAddress)
+      const isTokenInToBeLearnedTokens = tobeLearnedTokens.includes(lowercaseAddress)
```

Comment:
I don't think this check is ok. addresses in `toBeLearnedTokens` are checksummed and `lowercaseAddress` is lowercase